druid-shell/src/platform/windows/window.rs

```diff
@@ -1293,7 +1305,11 @@ impl WindowBuilder {
             };
             win.wndproc.connect(&handle, state);
 
-            let mut dwStyle = WS_OVERLAPPEDWINDOW;
+            let mut dwStyle = if self.transparent {
+                WS_POPUP
```

Comment:
not every transparent window is a popup!!

Reply:
I removed that, new way of doing transparency doesn't require WS_POPUP, old way required that.